caom2-compute/src/test/java/ca/nrc/cadc/caom2/compute/ComputeDataGenerator.java

```diff
@@ -129,12 +129,39 @@ TemporalWCS mkGoodTemporalWCS() {
 
     TemporalWCS mkBadTemporalWCSCunit() {
 
+        double px = 0.5;
+        double sx = 54321.0;
+        double nx = 200.0;
+        double ds = 0.01;
+        RefCoord c1 = new RefCoord(px, sx);
```

Comment:
Some of the variables above are not used but help to make the test confusing. Instead, just create two RefCoord instances with constants, say:

(0.5, 54321.0) and (100.5, 54321.5)

which basically means there are 100 pixels that span 0.5 days. You want to maker sure the time axis is otherwise valid so the invalid unit is causing the fail and it helps to make the values easy to understand.